lib/__tests__/plugins.test.js

```diff
@@ -108,6 +117,37 @@ it('config with its own plugins extending another config that invokes a plugin w
 	expect(result.warnings()[0].node).toBeTruthy();
 });
 
+it('plugin with CJS require()', async () => {
+	const result = await processorPluginRequire.process(cssWithFoo, { from: undefined });
+
+	expect(result.warnings()).toHaveLength(2);
+	expect(result.warnings()[0].text).toBe('found .foo (plugin/warn-about-foo)');
```

Comment:
```suggestion
	expect(result.warnings()[0].text).toBe('found .foo (plugin/warn-about-foo)');
	expect(result.warnings()[1].text).toBe('Unexpected empty block (block-no-empty)');
```

Reply:
Thanks for the suggestions, have updated with the same 👍🏽